hack/windows/Prepare-Node.ps1

```diff
@@ -1,37 +1,30 @@
 <#
 .SYNOPSIS
-(Test Only)Assists with preparing a Windows VM prior to calling kubeadm join
+Assists with preparing a Windows VM prior to calling kubeadm join
 
 .DESCRIPTION
-This script is only used for test to assist with joining a Windows node to a cluster.
-For production environment please follow the antrea windows installation guide and use Kubernetes official PrepareNode.ps1
-script from the sig-windows-tools repo release page: https://github.com/kubernetes-sigs/sig-windows-tools/releases
+This script assists with joining a Windows node to a cluster.
 - Downloads Kubernetes binaries (kubelet, kubeadm) at the version specified
-- Registers wins as a service in order to run kube-proxy and antrea-agent as DaemonSets.
+- Registers wins as a service in order to run kube-proxy and cni as DaemonSets.
```

Comment:
It doesn't seem the change is more correct. cni is more abstract, why is this change?